docs/synthetics/status-pages.md

```diff
@@ -0,0 +1,37 @@
+title: Status Pages
+description: Communicate the status of your services to your customers using status pages
+
+Using Sematext Synthetics you can create public status pages to share the status and response time of your services to your customers.
+
+## Create Status Page
+
+You can create multiple status pages under your account. Navigate to the `Synthetics -> Status Pages` screen to create a status page. 
+
+![Create Status Page](../images/synthetics/status-page-create.png)
+
+The following fields are required to create a status page:
+
+* **Monitors** - List of monitors to be included in the status page. All the active monitors from Apps for which the account has OWNER or ADMIN role can be included in the status page. A monitor can be included in multiple status pages.
+* **Name** - Unique name for the status page.
+* **Header Name** - Company name to be displayed in the status page header.
+* **Header Logo** - URL of the company logo to be displayed in the status page header. An image with a height of 40px and white background is recommended.
+* **Response Time Aggregation** - Aggregation to apply for Response Time chart. The response time is aggregated in 1 hour buckets.
+* **Custom domain** - Optionally you can host the status page in your own custom domain.
```

Comment:
in your own ==> under your own?  using your own?  with your own?

Reply:
Done